sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -214,6 +210,56 @@ def get_queue_receiver(self, queue_name, **kwargs):
             **kwargs
         )
 
+    def get_queue_deadletter_receiver(self, queue_name, **kwargs):
+        # type: (str, Any) -> ServiceBusReceiver
+        """Get ServiceBusReceiver for the sub Dead Letter Queue of the specific Queue.
+
+        :param str queue_name: The path of specific Service Bus Queue the client connects to.
+        :keyword mode: The mode with which messages will be retrieved from the entity. The two options
+         are PeekLock and ReceiveAndDelete. Messages received with PeekLock must be settled within a given
+         lock period before they will be removed from the queue. Messages received with ReceiveAndDelete
+         will be immediately removed from the queue, and cannot be subsequently rejected or re-received if
+         the client fails to process the message. The default mode is PeekLock.
+        :paramtype mode: ~azure.servicebus.ReceiveSettleMode
+        :keyword int prefetch: The maximum number of messages to cache with each request to the service.
+         The default value is 0, meaning messages will be received from the service and processed
+         one at a time. Increasing this value will improve message throughput performance but increase
+         the change that messages will expire while they are cached if they're not processed fast enough.
+        :keyword float idle_timeout: The timeout in seconds between received messages after which the receiver will
+         automatically shutdown. The default value is 0, meaning no timeout.
+        :keyword int retry_total: The total number of attempts to redo a failed operation when an error occurs.
+         Default value is 3.
+        :keyword float retry_backoff_factor: Delta back-off internal in the unit of second between retries.
+         Default value is 0.8.
+        :keyword float retry_backoff_max: Maximum back-off interval in the unit of second. Default value is 120.
+        :keyword bool transfer_deadletter: Whether to connect to the transfer deadletter queue, or the standard
```

Comment:
So when I consumed SB I would see this option and always brush it off because "I don't know what this is but I don't THINK I need to care"; so if we can give slightly more detail (since it is very esoteric) it might be useful.

perhaps adding to the end (before default is fault) "The Transfer Deadletter Queue is a distinct deadletter queue specifically for failures when routing between queues or topics using the ForwardTo or SendVia functionality."  A bit verbose, feel free to shorten, but I think you get the gist of what I'm contemplating.

Reply:
emmm, I tried to abstract it a bit, how does it look to you now?

feel free to give git suggestions :)

Reply:
Sounds very clear.++